ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-authorization.j2

```diff
@@ -1,25 +1,38 @@
-{% if aaa_authorization is defined and aaa_authorization is not none %}
+{% if aaa_authorization is arista.avd.defined %}
 
 ## AAA Authorization
 
 ### AAA Authorization Summary
 
 | Type | User Stores |
 | ---- | ----------- |
-{%     if aaa_authorization.exec.default is defined %}
+{%     if aaa_authorization.exec.default is arista.avd.defined %}
 | Exec | {{ aaa_authorization.exec.default }} |
 {%     endif %}
-
-{%     if aaa_authorization.config_commands is defined and aaa_authorization.config_commands == true %}
+{%     if aaa_authorization.config_commands is arista.avd.defined(true) %}
 Authorization for configuration commands is enabled.
 {%     else %}
 Authorization for configuration commands is disabled.
 {%     endif %}
+{%     if aaa_authorization.commands.privilege is arista.avd.defined or aaa_authorization.commands.all_default is arista.avd.defined %}
 
-{%     if aaa_authorization.serial_console is defined and aaa_authorization.serial_console == true %}
-Authorization for serial console is enabled.
+### AAA Authorization Privilege Levels Summary
 
+| Privilege Level | Group |
+| --------------  | ----- |
+{%         if aaa_authorization.commands.all_default is arista.avd.defined %}
+| all | {{ aaa_authorization.commands.all_default }} |
+{%         endif %}
+{%         if aaa_authorization.commands.privilege is arista.avd.defined %}
+{%             for command_levels in aaa_authorization.commands.privilege | arista.avd.natural_sort %}
```

Comment:
command_level is one item in ```Please add blank line after this if condition```

So change it to singular form, **command_levels** should be changed to **command_level**